base/special/trig.jl

```diff
@@ -80,6 +82,80 @@ end
 sin_kernel(x::Real) = sin(x)
 cos_kernel(x::Real) = cos(x)
 
+# acos methods
+ACOS_X_MIN_THRESHOLD(::Type{Float32}) = 2.0f0^-26
+ACOS_X_MIN_THRESHOLD(::Type{Float64}) = 2.0^-57
+PIO2_HI(::Type{Float32}) = 1.5707962513f+00
+PIO2_LO(::Type{Float32}) = 7.5497894159f-08
+PIO2_HI(::Type{Float64}) = 1.57079632679489655800e+00
+PIO2_LO(::Type{Float64}) = 6.12323399573676603587e-17
+ACOS_PI(::Type{Float32}) = 3.1415925026f+00
+ACOS_PI(::Type{Float64}) = 3.14159265358979311600e+00
+ACOS_CORRECT_LOWWORD(::Type{Float32}, x) = reinterpret(Float32, (reinterpret(UInt32, x)&0xfffff000))
+ACOS_CORRECT_LOWWORD(::Type{Float64}, x) = reinterpret(Float64, (unsafe_trunc(UInt64, highword(x)) << 32) | UInt32(0))
+
+acos_p(z::Float32) =
```

Comment:
these `acos_p` and `acos_q` should simply be renamed `arc_p` and `arc_q` as they are the same as for the `asin` implementation.